internal/graph/lookupresources3.go

```diff
@@ -1399,7 +1399,7 @@ func mustDatastoreIndexToString(index datastoreIndex) (string, error) {
 		return dsIndexPrefix + index.chunkID, nil
 	}
 
-	return dsIndexPrefix + index.chunkID + ":" + (*index.dbCursor).String(), nil
+	return dsIndexPrefix + index.chunkID + ":" + index.dbCursor.RelationshipReference.String(), nil
```

Comment:
I'm not entirely sure how this deref worked previously - this both makes things a little clearer and also satisfies gocritic.

Reply:
I thought this was the test failure but it's not 🤔